tests/RobotFramework/tests/cumulocity/self-update/tedge_self_update.robot

```diff
@@ -142,8 +136,15 @@ Create Local Repository
     Execute Command    cd /opt/repository/local && dpkg-scanpackages -m . > Packages
     Execute Command    cmd=echo 'deb [trusted=yes] file:/opt/repository/local /' > /etc/apt/sources.list.d/tedge-local.list
 
-Install Packages
-    [Arguments]    ${packages_dir}=/setup/packages
-
-    Execute Command    apt-get update && apt-get install -y --no-install-recommends mosquitto
-    Execute Command    cd ${packages_dir} && dpkg -i tedge_*.deb && dpkg -i tedge*mapper*.deb && dpkg -i tedge*agent*.deb && dpkg -i tedge*watchdog*.deb && dpkg -i tedge*apt*plugin*.deb
+Install fixed thin-edge.io version
+    [Documentation]    APT Package pinning is used to install a fixed version as specifying a version via `apt install tedge-full=1.0.0`
+            ...        does not work as tedge-full is a virtual packages and the version does not get propogated to the package dependencies.
```

Comment:
```suggestion
            ...        does not work as tedge-full is a virtual packages and the version does not get propagated to the package dependencies.
```

Reply:
This comment has now been removed because an alternative approach which avoids using package pinning was used. 3f1d1ffd1670f0c560d5cd77dc7d39a8d5a52c04